pepper-apis/src/appengine/DataDonationPlatform.yaml

```diff
@@ -1,17 +1,11 @@
 service: pepper-backend
 runtime: java11
-instance_class: F4
+instance_class: B4
+
+basic_scaling:
+  max_instances: 4
+  idle_timeout: 10m
 
```

Comment:
These are the only params available. I am assuming there will always be an instance running, but not sure yet. idle_timeout use to determine whether can be scaled down.